### PR TITLE
Automatically publish tags to Bower.

### PR DESCRIPTION
**Pull request by rwjblue**
/cc @chancancode

### CODE REVIEW

**Comment by chancancode** [comment]
TODO:

* [ ] include docs json in the auto published builds

**Comment by locks** [comment]
@chancancode is your TODO item a blocker or can we merge now and PR that later?

**Comment by locks** [comment]
@chancancode @rwjblue poke

**Comment by Serabe** [comment]
@chancancode @rwjblue ☝️

File: bin/publish_npm_and_bower [hunk 1]
```diff
@@ -40,20 +41,45 @@ case $TRAVIS_BRANCH in
...
 if [[ -n $BUILD_TYPE ]]; then
   CHANNEL=$BUILD_TYPE
   echo -e "CHANNEL OVERRIDE: ${CHANNEL}\n"
 fi
 
+#### NPM PUBLISHING
+if [[ "$TRAVIS_TAG" != "" ]]; then
+  echo $NPM_AUTH_TOKEN > ~/.npmrc
+  if [[ "$CHANNEL" == "release" ]]; then
+    echo "npm publish"
```
Comment (chancancode) [inline bin/publish_npm_and_bower]: We forgot to fix this

File: bin/publish_npm_and_bower [hunk 2]
```diff
@@ -40,20 +41,45 @@ case $TRAVIS_BRANCH in
...
+  else
+    echo "npm publish --tag $CHANNEL"
```
Comment (chancancode) [inline bin/publish_npm_and_bower]: ditto

**Pull request merged**